bedrock/firefox/templates/firefox/new/quantum/base.html

```diff
@@ -45,7 +45,7 @@
     {% block messages %}{% endblock %}
     <div class="content">
       <div class="header-logos">
-        <h2><a class="firefox" href="{{ url('firefox') }}">{{ high_res_img('logos/firefox/logo-quantum-wordmark-white.png', {'alt': 'Firefox', 'width': '133', 'height': '51'}) }}</a></h2>
+        <h2><a class="firefox" href="{{ url('firefox') }}">{{ high_res_img('logos/firefox/logo-quantum-wordmark-white.png', {'alt': 'Firefox', 'width': '133', 'height': '50'}) }}</a></h2>
```

Comment:
I needed a slightly bigger version of this image but didn't want to add yet another dupe logo so I updated the original but had to tweak the size here. This was the only other place it was used so far, but if other new pages also rely on this asset they may need to adjust sizing.

Reply:
Yes